PWGHF/HFC/TableProducer/correlatorDplusHadrons.cxx

```diff
@@ -290,6 +292,11 @@ struct HfCorrelatorDplusHadrons {
     registry.add("hPhiMcGen", "D+,Hadron particles - MC Gen", {HistType::kTH1F, {axisPhi}});
     registry.add("hMultFT0AMcGen", "D+,Hadron multiplicity FT0A - MC Gen", {HistType::kTH1F, {axisMultiplicity}});
     corrBinning = {{binsZVtx, binsMultiplicity}, true};
+    if (isPromptAnalysis) {
+      idxBdtScore = 1;
+    } else {
+      idxBdtScore = 2;
+    }
```

Comment:
Consider replacing it with a one-liner:
```suggestion
    idxBdtScore = isPromptAnalysis ? 1 : 2;
```

Reply:
I think this will not be required with the new modifications as you suggested in your next comment.